site/docs/components/autocomplete.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: Autocomplete
+navTitle: Autocomplete (not built)
+summaryParagraph: Autocomplete automatically completes typed user input with matching results from a larger data set. Autosuggest breaks beyond the input provided to suggest alternative, relevant answers.
+tags: ["Autosuggest", "Live filter", "Filters", "Search", "Top bar", "Text input"]
+needToKnow:
+- 'We separate Search Field into a separate component, because it may be used for live-filtering without showing autocompleted results.'
```

Comment:
I found this a bit hard to parse. Not clear what thing the search field has been separated from and where autocomplete fits into the picture. Maybe something like  "Search and autocomplete are separated into distinct components, because  search may be used in a way that doesn't show autocompleted results." ?